src/main/java/org/broadinstitute/hellbender/tools/walkers/SplitIntervals.java

```diff
@@ -108,9 +111,15 @@
             shortName = StandardArgumentDefinitions.OUTPUT_SHORT_NAME)
     public File outputDir;
 
+    @Argument(doc = "Prefix to use when writing interval files", fullName = INTERVAL_FILE_PREFIX_FULL_NAME, optional = true)
+    public String prefix = DEFAULT_PREFIX;
+
     @Argument(doc = "Extension to use when writing interval files", fullName = INTERVAL_FILE_EXTENSION_FULL_NAME, optional = true)
     public String extension = DEFAULT_EXTENSION;
 
+    @Argument(doc = "Number of digits to use when writing interval files", fullName = INTERVAL_NUMBER_OF_DIGITS_FULL_NAME, optional = true)
+    public int numDigits = DEFAULT_NUMBER_OF_DIGITS;
```

Comment:
Do some up-front validation on this argument value to make sure it's > 0, and throw a `UserException` if it's not.

Reply:
good catch.  I populated the `minValue` parameter for the `@Argument` annotation.